odl/solvers/util/callback.py

```diff
@@ -483,7 +492,17 @@ def __init__(self, *args, **kwargs):
         self.fig = kwargs.pop('fig', None)
         self.step = kwargs.pop('step', 1)
         self.saveto = kwargs.pop('saveto', None)
+        try:
+            self.saveto_formatter = self.saveto.format
+        except AttributeError:
+            self.saveto_formatter = self.saveto
```

Comment:
Well if they decide to fix it we can blacklist scipy 0.19 for the time being. Although I suspect that this issue (amazing it hasn't come up earlier) can potentially pop up at any place any time.
Otherwise I guess what we can only either stop using `newint` or `np.isscalar`.